### PR TITLE
Import window GUI added.

### PR DESCRIPTION
**Pull request by mateuszpach**
Import window GUI added. Icon added.

### CODE REVIEW

File: src/com/github/mtdrewski/GRAPH_moment/controller/Menu.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+    private Supplier<Stage> dialogFactory = () -> {
+        Stage dialog = new Stage();
+        dialog.initModality(Modality.APPLICATION_MODAL);
+        dialog.initOwner((Stage) menu.getScene().getWindow());
+        dialog.getIcons().add(new Image("icon.png"));
+        return dialog;
+    };
+
+    public void newProjectOnClick(ActionEvent event) throws Exception {
```
Comment (woocash2) [inline src/com/github/mtdrewski/GRAPH_moment/controller/Menu.java]: Does it throw just any Exception? (Maybe use more specific Exception mark)

Comment (mateuszpach) [reply src/com/github/mtdrewski/GRAPH_moment/controller/Menu.java]: It should probably be IOException, but so far I'm not sure if it should be thrown and not catch. Moreover newProjectOnClick isn't finished, so just ignore it for now.

Comment (woocash2) [reply src/com/github/mtdrewski/GRAPH_moment/controller/Menu.java]: ok, so just mark it as //TODO or something

File: src/com/github/mtdrewski/GRAPH_moment/view/graph_display.fxml [hunk 2]
```diff
@@ -1,8 +1,4 @@
-<?import javafx.scene.canvas.Canvas?>
```
Comment (woocash2) [inline src/com/github/mtdrewski/GRAPH_moment/view/graph_display.fxml:1]: Why does this appear here?
I deleted this stuff recently in my commit

File: src/com/github/mtdrewski/GRAPH_moment/controller/GraphDrawer.java [hunk 3]
```diff
@@ -27,7 +27,6 @@
 
 
     private Supplier<VertexCircle> vertexShapeFactory = () -> {
-
```
Comment (woocash2) [inline src/com/github/mtdrewski/GRAPH_moment/controller/GraphDrawer.java:30]: This cannot be forgiven...

**Pull request merged**